std/datetime/date.d

```diff
@@ -9986,12 +9986,11 @@ int cmpTimeUnits(string lhs, string rhs) @safe pure
     import std.exception : enforce;
     import std.format : format;
 
-    auto tstrings = timeStrings;
-    immutable indexOfLHS = countUntil(tstrings, lhs);
-    immutable indexOfRHS = countUntil(tstrings, rhs);
+    immutable indexOfLHS = countUntil(timeStrings, lhs);
+    immutable indexOfRHS = countUntil(timeStrings, rhs);
```

Comment:
Just curious, what was the reason for this change?

Reply:
No reason to make a second copy of the reference when countUntil takes it by value

Reply:
Fair enough.